lib/manager/argocd/__fixtures__/validApplication.yml

```diff
@@ -0,0 +1,38 @@
+---
+apiVersion: argoproj.io/v1alpha1
+kind: Application
+metadata:
+...
+spec:
+  source:
+    chart: kube-state-metrics
+    repoURL: https://kubernetes-charts.storage.googleapis.com
```

Comment:
This url is no longer accessible. Can we use the new working url for stabile charts?